dot/rpc/modules/author_integration_test.go

```diff
@@ -221,15 +220,8 @@ func TestAuthorModule_SubmitExtrinsic_AlreadyInPool(t *testing.T) {
 		integrationTestController.runtime, genesisHash, genesisHash, 0, "System.remark", []byte{})
 	extBytes := common.MustHexToBytes(extHex)
 
-	storageState := coremocks.NewMockStorageState(ctrl)
-	// should not call storage.TrieState
-	storageState.EXPECT().TrieState(nil).MaxTimes(0)
-	integrationTestController.storageState = storageState
-
-	net2test := coremocks.NewMockNetwork(ctrl)
-	// should not call network.GossipMessage
-	net2test.EXPECT().GossipMessage(nil).MaxTimes(0)
-	integrationTestController.network = net2test
+	integrationTestController.storageState = &state.StorageState{}
+	integrationTestController.network = NewMockNetwork(ctrl)
```

Comment:
nit let's pass it `nil` to highlight it's meant to just be there not doing any mock stuff

```suggestion
	integrationTestController.network = NewMockNetwork(nil)
```